### PR TITLE
Unit test for format conversion

### PR DESCRIPTION
**Pull request by namnguyen20999**
image

### CODE REVIEW

File: frontend/taipy-gui/src/utils/formatConversion.ts [hunk 1]
```diff
@@ -44,34 +44,36 @@ const sprintfParse = (fmt?: string): (string | { placeholder: string; })[] => {
...
         }
 
         if (match) {
             _fmt = _fmt.substring(match[0].length);
         }
     }
-
     return parseTree;
-}
+};
```
Comment (FabienLelaquais) [inline frontend/taipy-gui/src/utils/formatConversion.ts:65]: No linter complains about this additional ';'?

File: frontend/taipy-gui/src/utils/formatConversion.ts [hunk 2]
```diff
@@ -108,9 +108,9 @@ export const sprintfToD3Converter = (fmt?: string): string => {
 export const extractPrefix = (fmt?: string): string => {
     if (!fmt) return "";
     const sprintfFmtArr = sprintfParse(fmt);
-    const objectIndex = sprintfFmtArr.findIndex((element) => typeof element === 'object');
-    return sprintfFmtArr.slice(0, objectIndex).join('');
-}
+    const objectIndex = sprintfFmtArr.findIndex((element) => typeof element === "object");
+    return sprintfFmtArr.slice(0, objectIndex).join("");
+};
```
Comment (FabienLelaquais) [inline frontend/taipy-gui/src/utils/formatConversion.ts:113]: Same

**Review by FabienLelaquais — APPROVED** [review]
👍👍

**Review by FredLL-Avaiga — APPROVED** [review]

File: frontend/taipy-gui/src/utils/formatConversion.ts (hunk 1 above)
Comment (namnguyen20999) [reply frontend/taipy-gui/src/utils/formatConversion.ts:65]: nope, actually my prettier automatically put that additional ";" after a closing brace (}) for every objects, functions, etc.

Comment (namnguyen20999) [reply frontend/taipy-gui/src/utils/formatConversion.ts:65]: and the linter not complaining anything about that extra ";" as well

**Pull request merged**